src/main/java/net/coderbot/patchwork/manifest/forge/AccessTransformerEntry.java

```diff
@@ -0,0 +1,49 @@
+package net.coderbot.patchwork.manifest.forge;
+
+import net.fabricmc.tinyremapper.MemberInstance;
+import org.objectweb.asm.commons.Remapper;
+
+public class AccessTransformerEntry {
+	private String clazzName;
+	private String memberName;
+	private boolean memberIsField = false;
+	// todo inner class support
+	public AccessTransformerEntry(String clazzName,
+			String memberName,
+			Remapper voldeToOfficial,
+			Remapper officialToIntermediary) {
+		this.clazzName = clazzName;
+		this.memberName = memberName;
+		remap(voldeToOfficial);
+		remap(officialToIntermediary);
+		System.out.println(this.memberName);
+	}
+
+	private void remap(Remapper remapper /*remapper is easier to type*/) {
```

Comment:
I don't think this comment is necessary? It's a completely sensible name

Reply:
that's from when the variable was `voldeToIntermediaryRemapper`